provider/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.5.0'
+__version__ = '0.6.0'
```

Comment:
See for instance, [Semantic Versioning Spec, Item 4](http://semver.org/#spec-item-4).